src/util/Layer.js

```diff
@@ -12,8 +12,6 @@ const LayerUtil = {
    */
   getLayersBy (key, value, olMap) {
     if (!olMap) {
-      console.warn('No OL map passed to LayerUtil.getLayersBy - ' +
```

Comment:
Why is this removed?

Reply:
The tests gave too much output, but feel free to change.